typescript/cli/src/ism/read.ts

```diff
@@ -19,19 +19,32 @@ export async function readIsmConfig({
   address: Address;
   out?: string;
 }): Promise<void> {
-  if (context.multiProvider.getProtocol(chain) === ProtocolType.Ethereum) {
-    const ismReader = new EvmIsmReader(context.multiProvider, chain);
-    const config = await ismReader.deriveIsmConfig(address);
-    const stringConfig = stringifyObject(config, resolveFileFormat(out), 2);
-    if (!out) {
-      logBlue(`ISM Config at ${address} on ${chain}:`);
-      log(stringConfig);
-    } else {
-      writeFileAtPath(out, stringConfig + '\n');
-      logBlue(`ISM Config written to ${out}.`);
+  switch (context.multiProvider.getProtocol(chain)) {
+    case ProtocolType.Ethereum: {
+      const ismReader = new EvmIsmReader(context.multiProvider, chain);
+      const config = await ismReader.deriveIsmConfig(address);
+      const stringConfig = stringifyObject(config, resolveFileFormat(out), 2);
+      if (!out) {
+        logBlue(`ISM Config at ${address} on ${chain}:`);
+        log(stringConfig);
+      } else {
+        writeFileAtPath(out, stringConfig + '\n');
+        logBlue(`ISM Config written to ${out}.`);
+      }
```

Comment:
1. Instead of duplicating this code block in both switch branches, can't we define a `config` variable outside of the switch and after it is assigned in one of the 2 branches we have this code block at the end of this function? 
2. Can we refrain from using switch statements if we only have 2 cases? I think using `if ... else ...` in this case is perfectly valid